exercism/pangram/pangram.js

```diff
@@ -0,0 +1,27 @@
+function Pangram( str ) {
+  this.str = str,
+  this.asciiLookup = {
+    a: 0, b: 0, c: 0, d: 0, e: 0, f: 0, g: 0, h: 0, i: 0, j: 0, k: 0, l: 0, m: 0,
+    n: 0, o: 0, p: 0, q: 0, r: 0, s: 0, t: 0, u: 0, v: 0, w: 0, x: 0, y: 0, z: 0
+  }
+}
```

Comment:
I don't think this object has to be created on every instance of every instatiated object. I would rather see it as a variable set inside the program of have it imported from another file inside of your `isPanagram()` function.